templates/Mixing_Score.ipynb

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
<p>Does cold threshold apply to target, reference, or both? Should give a description for what this is.</p><p> </p><p>I think having a simple dist plot of the scores would be nice. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
applies to the reference population